saechimdaeki/boj1644소수의연속합.java

```diff
@@ -0,0 +1,33 @@
+import java.io.*;
+import java.util.*;
+import java.util.stream.IntStream;
+
+public class boj1644소수의연속합 {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

Reply:
This code is an algorithm code from a site called Baek Jun.
The java class on the site must be a class named Main unconditionally, and it is kept when submitted.

However, this repository is simply for viewing the algorithm-solving logic and does not care about such matters to classify the problem.

Reply:
and i updated comment code not code file

Reply:
I simply didn't care about the class name, but thanks for the feedback, I will definitely do that next time!